src/spaces/utilities.jl

```diff
@@ -1,5 +1,4 @@
-export edistance,
-    walk!, North, South, East, West, NorthEast, NorthWest, SouthEast, SouthWest
+export edistance, walk!
```

Comment:
Why do we export `edistance`?

Reply:
Since it respects the PBCs. Users don't need to care about them when calculating distance. It's used in `examples/flock.jl` as an example.

Reply:
oh okay sorry didn't remember it